read-fonts/src/tables/fvar.rs

```diff
@@ -21,22 +21,37 @@ impl<'a> Fvar<'a> {
 
 impl VariationAxisRecord {
     /// Returns a normalized coordinate for the given value.
-    pub fn normalize(&self, mut value: Fixed) -> Fixed {
-        use core::cmp::Ordering::*;
-        let min_value = self.min_value();
-        let default_value = self.default_value();
-        // Make sure max is >= min to avoid potential panic in clamp.
-        let max_value = self.max_value().max(min_value);
-        value = value.clamp(min_value, max_value);
-        value = match value.cmp(&default_value) {
-            Less => -((default_value - value) / (default_value - min_value)),
-            Greater => (value - default_value) / (max_value - default_value),
-            Equal => Fixed::ZERO,
-        };
-        value.clamp(-Fixed::ONE, Fixed::ONE)
+    pub fn normalize(&self, value: Fixed) -> Fixed {
+        normalize_coordinate(
+            value,
+            self.min_value(),
+            self.default_value(),
+            self.max_value(),
+        )
     }
 }
 
+/// Compute the normalized coordinate for a given input coordinate and axis min/max/default values.
+///
+/// This impl is shared with `write-fonts`.
+pub fn normalize_coordinate(
```

Comment:
I've marked this as `#[doc(hidden)]` and clarified the docs a bit to explain that it's only exposed so that the logic can be reused between read/write.